webapp/locales/en.json

```diff
@@ -158,6 +158,28 @@
       "labelBio": "About You",
       "success": "Your data was successfully updated!"
     },
+    "email": {
+      "validation": {
+        "same-email": "Must be different from your current E-Mail address"
+      },
+      "name": "Your E-Mail",
+      "labelEmail": "Change your E-Mail address",
+      "labelNewEmail": "New E-Mail Address",
+      "labelNonce": "Enter your code",
+      "success": "A new E-Mail address has been registered.",
+      "submitted": "An email to verify your address has been sent to <b>{email}</b>.",
+      "change-successful": "Your E-Mail address has been changed successfully.",
+      "verification-error": {
+        "message": "Your E-Mail could not be changed.",
+        "explanation": "This can have different causes:",
+        "reason": {
+          "invalid-nonce": "Is the confirmation code invalid?",
+          "no-email-request": "You haven't requested a change of your email address at all?",
+          "email-address-taken": "Has the email been assigned to another user in the meantime?"
+        },
+        "support": "If the problem persists, please contact us by e-mail at"
```

Comment:
I think we didn't decide... @Tirokk asked it to be changed in your `PR` cause it's what _he_ prefers, and since no one else cared enough to point it out, it's what we went with. I prefer the non-hyphenated version personally. While `e-mail` is still more commonly used than `email`, it's on the downturn, so if we wanna keep up with the times, maybe we should be cool and use `email` :stuck_out_tongue_closed_eyes: 
https://writingexplained.org/e-mail-or-email

Reply:
So how should I change it? `email` now?

Reply:
In German it seems to `E-Mail` and nothing else: https://www.duden.de/sprachwissen/sprachratgeber/Schreibung-von-E-Mail

Reply:
I say just leave it, like I said, it's not something I would ask to be changed. 
Sorry, I just couldn't help myself when @alina-beck said we decided... `e-mail` is perfectly ok, there seem to be stricter rules in German than English and as long as it's not incorrect, we can follow the stricter rules to have some continuity between the two.

Reply:
I really don't mind if we go with `email` or `e-mail` in English! (Personally I've only been using `email` so far...) But I think it would be great if we could agree on one and then stick to it in all user-facing communication.

It's not a big issue and most people won't notice, I guess, but some (language :nerd_face: like me) might and it's just overall smoother and cleaner. :wink:

Since I don't have a strong opinion I just went with what @Tirokk suggested last time. But what do you say @mattwr18? Would `email` be better? Let's decide! :smile:

Reply:
I wouldn't say it would be better, I don't think it really matters one way or the other, and I completely understand for user facing communication it's great to be consistent. It's a little funny to me cause all of our `keys` in the `locale` files we spell it `email` not `e-mail`, but those are not user facing.
I'm happy to use `e-mail` for the reasons I stated above :point_up: 
Now, we can probably freely say we decided :rofl: